### PR TITLE
[v1.13] Backport print-chart-version script for dev Helm chart push

### PR DESCRIPTION
**Pull request by joestringer**
[ upstream commit d90803cec8e7a1508558b183600cde7dcbdd719f ]
[ upstream commit 2c367de92d1825bf2ab80de2fd4e848ee2e28c46 ]
[ Backporter's notes:
  - Removed github workflow. Workflow operates from main branch.
  - Applied /usr/bin/env hunk to the script (was treewide). ]

Related: #25205

```upstream-prs
$ for pr in 25205; do contrib/backporting/set-labels.py $pr done v1.13; done
```

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]
let's get https://github.com/cilium/cilium/pull/26086 merged first. otherwise the push chart workflow will start failing on v1.13 branch.

edit: oh it's the other way around. this one needs to go first so that the script exists 👀

**Comment by michi-covalent** [comment]
✅ confirmed the chart version here: https://github.com/cilium/cilium/actions/runs/5224577019 (`1.13.3-dev.35-pr-joe-version-script-v1-13-e76d38d1d6`)

https://quay.io/repository/cilium-charts-dev/cilium?tab=tags

no need for the full CI run. let's just wait for the basic CI to finish before merging.

**Review by chancez — APPROVED** [review]

**Pull request merged**
